depmapomics/qc/rna.py

```diff
@@ -1,10 +1,11 @@
+from depmapomics import constants
 import sys
 
 import pandas as pd
 from genepy import rna
 from depmapomics import terra
 import numpy as np
-from depmapomics.config import RNASEQC_THRESHOLDS_FAILED, RNASEQC_THRESHOLDS_LOWQUAL
+from depmapomics import constants
```

Comment:
here it is actually used, but import is duplicated

Reply:
removed